Sources/VCIClient/errors/DownloadFailedError.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 
 public enum DownloadFailedError: Error {
```

Comment:
can we make it non public, if we are not accessing it outside

Reply:
Modified the accessibility here